src/com/goide/project/GoLibrariesService.java

```diff
@@ -38,7 +38,7 @@
 import java.util.Set;
 
 public abstract class GoLibrariesService<T extends GoLibrariesState> extends SimpleModificationTracker implements PersistentStateComponent<T> {
-  public static final Topic<LibrariesListener> LIBRARIES_TOPIC = new Topic<LibrariesListener>("libraries changes", LibrariesListener.class);
+  public static final Topic<LibrariesListener> LIBRARIES_TOPIC = new Topic<LibrariesListener>("Go packages changes", LibrariesListener.class);
```

Comment:
I've changed this as the argument name is `displayName` and I thought this might be actually used to display something to the end user. If it's not, I'll revert it.